gh.json

```diff
@@ -1,22 +1,40 @@
 {
-    "version": "0.11.1",
+    "version": "0.12.0",
     "architecture": {
         "32bit": {
-            "url": "https://github.com/cli/cli/releases/download/v0.11.1/gh_0.11.1_windows_386.zip",
+            "url": "https://github.com/cli/cli/releases/download/v0.12.0/gh_0.12.0_windows_386.zip",
             "bin": [
                 "bin/gh.exe"
             ],
-            "hash": "aae2cfeead222eb504d527033c95e853a0b1eb685009a5bcc8bca20af98eb8bb"
+            "hash": "ab9ff7487e3b684819a0f67a7522d29271dfc4c3f227de7cf00bfe31ac02c698"
         },
         "64bit": {
-            "url": "https://github.com/cli/cli/releases/download/v0.11.1/gh_0.11.1_windows_amd64.zip",
+            "url": "https://github.com/cli/cli/releases/download/v0.12.0/gh_0.12.0_windows_amd64.zip",
             "bin": [
                 "bin/gh.exe"
             ],
-            "hash": "0ff00a7d89d1b638a6f3cbce6a09902c5a3d1759ed28463971c9bc2e008caf63"
+            "hash": "6f79296f7c37121ee5853d79a1241eb7d3ba892d491953db1d375a498c4b84ad"
         }
     },
     "homepage": "https://github.com/cli/cli",
     "license": "MIT",
-    "description": "GitHub CLI"
-}
\ No newline at end of file
+    "description": "GitHub CLI",
+    "checkver": "github",
```

Comment:
Thanks for proposing this! I'm not sure if this is a good idea, since we now have two update checkers: one baked into the `gh` binary (you saw that one) and, with this, the performed by scoop as configured here. I don't think two update checkers running independently of each other would be a great experience for our users.

As a scoop user, would you prefer that scoop checks for updates, or the binary itself?

Reply:
It's great that the binary checks for updates and I think that should remain in place; however, it's not immediately actionable (it's passive), which is why I proposed this change. This autoupdate mechanism is more for scoop developers. As I understand it, and in my experience with scoop and writing manifests, this is really more of a tool owner (e.g. you) or power user tool. If you run bin\checkver.ps1 from the scoop binary folder, it actually updates the manifest for you if there are updates. It doesn't actually update the installed tool. But it makes it easy to update the manifest. This could be plugged into your release pipeline for cli/cli or, alternatively, have the pipeline generate the file each time (URLs are deterministic, and you generate sha256 hashes already since you populate them in a file).

This way, however, let's power users that know about bin\checkver.ps1 to actually update the manifest before you have a chance to update the manifest. Once you run bin\checkver.ps1, you can run `scoop update gh` and you're updated.

Reply:
Thanks for explaining!

The manifest in this repo is supposed to be kept always to date by our release automation. (This did not work with the latest release, as you've pointed out, but that was a bug.) If our manifest is guaranteed to always be up to date with the latest release, does running `bin\checkvers.ps1` even make sense then?

Reply:
Not much, no. If the impact is near-immediate anyway, there's really no reasons for power users to run checkver.ps1.